base/printf.jl

```diff
@@ -36,23 +36,24 @@ end
 ### printf format string parsing ###
 
 function parse(s::AbstractString)
-    # parse format string in to stings and format tuples
+    # parse format string into strings and format tuples
     list = []
     i = j = start(s)
+    j1 = 0 # invariant: j1 == prevind(s, j-1)
```

Comment:
@KlausC minor issue; if I understand it correctly the comment should read:
```
# invariant: j1 == prevind(s, j)
```

Reply:
you are right!